configuration/env-configuration.ts

```diff
@@ -0,0 +1,37 @@
+import {IConfiguration} from "./iconfiguration";
+
+export class EnvConfiguration implements IConfiguration {
+    private static envPrefix = "azureIot_";
+
+    /**
+     * Get the value associated with the passed key from the JSON
+     * object of user configurations.
+     *
+     * Keys stored as environment variables are stored in the environment
+     * with the prefix `azureIot_`.
+     *
+     * @param {string} key - Name of variable to get.
+     * @return {string} Value of configuration variable named by key.
+     */
+    public getString(key: string): string {
+        return process.env[EnvConfiguration.envPrefix + key];
+    }
+
+    /**
+     * Get the value associated with the passed key.
+     *
+     * Keys stored as environment variables are stored in the environment
+     * with the prefix `azureIot_`.
+     *
+     * @param {string} key - Name of variable to get.
+     * @return {any} Value of variable named by key.
+     */
+    public getAny(key: string): any {
```

Comment:
make generic get<T>(key: string): T...getString handles the failing case so let parse throw if it is called with a bad value